op-node/rollup/superchain.go

```diff
@@ -75,9 +75,9 @@ func LoadOPStackRollupConfig(chainID uint64) (*Config, error) {
 		// but since none of the superchain chains differ, it's not represented in the superchain-registry yet.
 		// This restriction on superchain-chains may change in the future.
 		// Test/Alt configurations can still load custom rollup-configs when necessary.
-		BlockTime:              2,
+		BlockTime:              chConfig.BlockTime,
 		MaxSequencerDrift:      600,
```

Comment:
Shall we change the comment just above? Maybe we mention that some but not all config fields are represented in the superchain-registry (yet).

Reply:
https://github.com/ethereum-optimism/optimism/pull/11172/commits/4ff66bbf08e4113bac056e5e63474fe9e162844a